src/plugins/mer/merbuildenginedetailswidget.cpp

```diff
@@ -64,6 +64,14 @@ MerBuildEngineDetailsWidget::MerBuildEngineDetailsWidget(QWidget *parent)
             + tr("Stop the virtual machine to unlock this field for editing.")
             + QLatin1String("</font>"));
 
+    m_ui->sshPortOccupiedLabel->setPixmap(Utils::Icons::CRITICAL.pixmap());
```

Comment:
WARNING would be more appropriate I think. We say the port "may" be already in use, anyway.

Reply:
Fixed.